src/component/feature/FeatureView/FeatureView.tsx

```diff
@@ -132,127 +132,117 @@ const FeatureView = () => {
         );
     };
 
+    if (error) {
```

Comment:
Doesn't this actually do exactly the same as conditionally render? The point is that we only want to show this if we get a 404 not found error back from the API.